exercicios/exercicio-alunas/vanessa-osorio/script.js

```diff
@@ -0,0 +1,24 @@
+
+
+let nome = prompt('Digite seu nome: ');
+let nota1 = Number(prompt('Digite a nota 1'));
+let nota2 = Number(prompt('Digite a nota 2'));
+let nota3 = Number(prompt('Digite a nota 3'));
+let media = (Number(nota1) + Number(nota2) + Number(nota3)) / 3;
+
+console.log(media);
+if (media >= 6.7) {
+    media = Math.ceil(media).toFixed(1);
+    alert(`Parabéns, ${nome} foi aprovado(a) com média ${media} , aproveite suas férias!`);
+} else if (media <= 6.8) {
```

Comment:
@vaosorio aqui a sintaxe é apenas else {}, porque se a media NÃO for  >= 6.7 o código já vai ser rodado dentro desse else.